src/toolchain/export-web.md

```diff
@@ -133,16 +133,22 @@ With the proposed initial configuration from ["Project configuration"](#project-
 your extension would break. If you'd like your extension to support builds without multi-threading as well to avoid this problem,
 you will need to update your build setup in one of the two following ways.
 
-Note that earlier versions of emscripten expected `link-args=-sUSE_PTHREADS=1` instead of `link-args=-pthread`, but this flag has
+```admonish note
+Ensure you're using recommended and recent versions of emscripten (3.1.x) and nightly Rust (at least Rust 1.85 is recommended).
```

Comment:
Emscripten is already at version 4, "recent" may be misleading here. Especially if 4.x is not mentioned, this sounds like already outdated advice.

Weren't there some specific versions (or a range) that was deemed "good" in our last discussion? I recently tested with `emcc 3.1.62`...

Reply:
You're right. I've made the message more explicit.